libnopegl/src/utils.c

```diff
@@ -44,6 +44,22 @@
 #include "utils.h"
 #include "pthread_compat.h"
 
+void ngli_rc_ref(struct ngli_rc *s)
```

Comment:
`void *ngli_rc_ref` and return `s` so that we can have: `s_priv->attachments_refs[s_priv->nb_attachments] = NGLI_RC_REF(attachment->attachment);` later in this PR?

Reply:
Done.